packages/access-api/test/ucan.test.js

```diff
@@ -75,26 +75,6 @@ describe('ucan', function () {
     ])
   })
 
-  test('should fail with bad service audience', async function () {
```

Comment:
This tests a part of the api that isn't important.
I also think after https://github.com/web3-storage/ucanto/pull/257/files that ucanto isn't doing this checking for us.

Reply:
It moved that check into the `provider(handler)` decorator.